integrations/docker/images/chip-build-vscode/Dockerfile

```diff
@@ -75,7 +75,7 @@ ENV IDF_PATH=/opt/espressif/esp-idf/
 ENV IDF_TOOLS_PATH=/opt/espressif/tools
 ENV IMX_SDK_ROOT=/opt/fsl-imx-xwayland/5.10-hardknott/
 ENV NRF5_TOOLS_ROOT=/opt/NordicSemiconductor/nRF5_tools
-ENV NXP_K32W061_SDK_ROOT=/opt/sdk/sdks
+ENV NXP_K32W0_SDK_ROOT=/opt/sdk
```

Comment:
Image 87 also failed for me with:

```
...
Step 31/60 : COPY --from=ameba /opt/ameba /opt/ameba
 ---> 82bb8c2f47ef
Step 32/60 : COPY --from=k32w /opt/sdk/sdks /opt/sdk/sdks
COPY failed: stat opt/sdk/sdks: file does not exist
```

I think we need to adjust more things here.

Reply:
Line 44 seems like it also needs adjusting.

Also /opt/sdk seems very generic ... 'sdk of what'? Why is this not /opt/kw32_sdk or something similar?